### PR TITLE
Borrow tabs

### PR DESCRIPTION
**Pull request by Meilin-Lyu**
--Create ui for borrow page
--There will be status setting in requested page
--Basic structure and mock data are same as library page
--Browse page will take up a different branch

### CODE REVIEW

File: app/src/main/java/com/cmput301f20t21/bookfriends/ui/borrow/AcceptedListAdapter.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import java.util.ArrayList;
+
+public class AcceptedListAdapter extends BaseBookListAdapter {
+    public AcceptedListAdapter(ArrayList<Book> books) {
+        super(books);
+    }
+
+    @NonNull
+    @Override
+    public com.cmput301f20t21.bookfriends.ui.borrow.AcceptedListAdapter.ViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
```
Comment (zhpeng811) [inline app/src/main/java/com/cmput301f20t21/bookfriends/ui/borrow/AcceptedListAdapter.java:25]: I think just AcceptedListAdapter.ViewHolder will work here

File: app/src/main/java/com/cmput301f20t21/bookfriends/ui/borrow/BorrowTabCollectionAdapter.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
+package com.cmput301f20t21.bookfriends.ui.borrow;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.Fragment;
+import androidx.viewpager2.adapter.FragmentStateAdapter;
+
+public class BorrowTabCollectionAdapter extends FragmentStateAdapter {
+    public enum TabType {
+        Requested_BOOKS,
+        Accepted_BOOKS
```
Comment (zhpeng811) [inline app/src/main/java/com/cmput301f20t21/bookfriends/ui/borrow/BorrowTabCollectionAdapter.java:10]: enum variables should be all captialized

**Comment by Meilin-Lyu** [comment]
borrow
accept

File: app/build.gradle [hunk 3]
```diff
@@ -49,6 +49,7 @@ dependencies {
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
     implementation 'androidx.lifecycle:lifecycle-livedata-ktx:2.2.0'
     implementation 'androidx.lifecycle:lifecycle-viewmodel-ktx:2.2.0'
+    implementation 'androidx.mediarouter:mediarouter:1.2.0'
```
[the remark this answers is not in the document]
Comment (Meilin-Lyu) [reply app/build.gradle]: you are right

**Review by zhpeng811 — APPROVED** [review]
lgtm

**Review by c25vdw — APPROVED** [review]
lgtm

**Pull request merged**